contents/tutorials/property-filter.md

```diff
@@ -9,83 +9,124 @@ tags:
-
-![PostHog Property Filter](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/property-filter/property-filter-tutorial.png)
-
-You can reorder the app chain simply by selecting 'Edit Order' and dragging the apps to run in any order you want. Apps at the top of the list run first, while those at the bottom run last. 
-
-# 3. Configure the filter to remove selected GeoIP info
-
-This app sets all specified properties on ingested events to `null`, effectively preventing PostHog from collecting information you do not want it to use.
-
-To configure the app to remove selected properties, simply select the blue gear icon and enter the properties you wish to remove.
-
-![PostHog Property Filter](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/property-filter/property-filter-tutorial-2.png)
-
-> **Note:** The GeoIP transformation adds event and person properties (`$set` and `$set_once`), so you'll want to filter those out as well. [See below](#event-and-person-properties).
-
-## The full list of GeoIP properties
-
-...at time of writing are
-
-* $geoip_city_name
-* $geoip_country_name
-* $geoip_country_code
-* $geoip_continent_name
-* $geoip_continent_code
-* $geoip_postal_code
-* $geoip_latitude
-* $geoip_longitude
-* $geoip_time_zone
-* $geoip_subdivision_1_name
-* $geoip_subdivision_1_code
-* $geoip_subdivision_2_name
-* $geoip_subdivision_2_code
-* $geoip_subdivision_3_code
-* $geoip_subdivision_3_name
-
-You can [check the current list in the source code for the app.](https://github.com/PostHog/posthog-plugin-geoip)
-
-## Event and Person properties
-
-The GeoIP app sets [person properties using `$set` and `$set_once`](https://posthog.com/docs/integrate/user-properties). If you want to drop those properties prefix the geoip property name with either `$set` or `$set_once`.
-
-For example to ensure `$geoip_cityname` is _never_ stored on either events or persons you would configure:
-
-* $geoip_city_name (the event property)
-* $set.$geoip_city_name (the person proprty)
-* $set_once.$initial_geoip_city_name (the initial and never updated person property)
-
-## A working example
-
-Below is the full configuration Witty Works uses to filter out unwanted data before it is written to PostHog's event log:
-
-```$geoip_city_name,$geoip_longitude,$geoip_latitude,$ip,$geoip_postal_code,$current_url,$performance_raw,$referrer,$initial_referrer,$pathname```
-
-<NewsletterForm />
\ No newline at end of file
+When collecting data with PostHog, you may sometimes want to avoid collecting certain properties for privacy reasons. For example, you may want to ensure you never collect specific kind of data such as IP addresses or exact locations.
```

Comment:
```suggestion
When collecting data with PostHog, you may sometimes want to avoid collecting certain properties for privacy reasons. For example, you may want to ensure you never collect data like IP addresses or exact locations.
```